action.yml

```diff
@@ -76,6 +76,9 @@ inputs:
     description: 'Time in seconds, after which token expires'
     required: false
     default: 3600
+  secretEncoding:
+    description: 'Encoding of the secret value. Can be "base64", "hex", "utf8".'
```

Comment:
When I first read this, I thought it would encode the secret value. I see that at actually decodes based on the encoding set here. Perhaps we can improve the description to make that more obvious?

Reply:
This is true.  Reading it again now I can see how it is fairly confusing.  Will work on making that clearer